.travis.yml

```diff
@@ -0,0 +1,65 @@
+sudo: false
+dist: trusty
+notifications:
+  irc:
+    channels:
+     - "chat.freenode.net#cockpit"
+    skip_join: true
+    use_notice: true
+    template:
+     - "%{message} (%{branch} - %{commit} : %{author}): %{build_url}"
+language: c
+compiler: gcc
+env:
+  global:
+   # The next declaration is the encrypted COVERITY_SCAN_TOKEN, created
+   #  via the "travis encrypt" command using the project repo's public key
+   - secure: "eZ6Idu0mdj8hovNPK0DL9MmUnzshblNtuXuvVMJ1zujdBJ09sZKmUDI44B2b8e64YPOTq8dTyP8GNmCBf8L3fw0knIfz0uxWBSMbqwLAJk5HkPQ3xE3vLJZK4Gs2np7IunTg91mgFcAkZo4gWNUOrB2jxv9bP0rZSummP8TV+c0="
```

Comment:
Is it safe/wise to expose the key like that? Wouldn't that mean that anyone could use that key to request scans, which could be mis-used for a DoS?